traveler-ios-ui/TravelerKitUI/ViewControllers/CancelOrderViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CancelOrderViewController.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-06-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+protocol CancelOrderViewControllerDelegate {
+    func orderCancelSucceed(with order:Order)
+}
+
+class CancelOrderViewController: UITableViewController {
```

Comment:
View controller naming convention is none verbal....`OrderCancellationViewController`. I would just opt for `CancellationViewController`